rclpy/rclpy/logging.py

```diff
@@ -25,11 +25,11 @@ class LoggingSeverity(IntEnum):
     This enum must match the one defined in rcutils/logging.h
     """
 
-    DEBUG = 0
-    INFO = 1
-    WARN = 2
-    ERROR = 3
-    FATAL = 4
+    DEBUG = 10
+    INFO = 20
+    WARN = 30
+    ERROR = 40
+    FATAL = 50
```

Comment:
Should we actually get the values from `rcutils/logging.h` rather than redefining them here?

Reply:
yes, we should, eventually. in my mind that would be done by moving the values into the `rcutils.logging` python module so we can include them here, and empy template them into rcutils' logging.h. Since there's a lot going on in https://github.com/ros2/rcutils/pull/57 already, I think it's best to put it off to a separate step

Reply:
Sounds good to me.